src/app/pages/AccountPage/index.tsx

```diff
@@ -35,32 +37,45 @@ import { ValidatorList } from '../StakingPage/Features/ValidatorList'
 import { AccountDetails } from './Features/AccountDetails'
 import { AccountSummary } from './Features/AccountSummary'
 
+const StyledNavItem = styled(NavLink)`
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding: ${({ theme }) => theme.global?.edgeSize?.small};
+
+  :hover {
+    background-color: ${({ theme }) => `${normalizeColor('background-contrast', theme)}`};
+  }
+
+  &.active {
+    background-color: ${({ theme }) => `${normalizeColor('background-back', theme)}`};
+  }
+`
+
 interface NavItemProps {
+  counter?: number
   label: string
   route: string
 }
 
-const NavItem = (props: NavItemProps) => {
-  const route = props.route
-  const label = props.label
-
-  const location = useLocation()
-  const isActive = route && route === location.pathname
-
+const NavItem = ({ counter, label, route }: NavItemProps) => {
   return (
-    <NavLink to={route}>
-      <Box
-        pad="xsmall"
-        // onClick={() => {}} hoverIndicator={{ color: 'background-oasis-blue' }}
-      >
+    <StyledNavItem exact to={route}>
+      <Text>{label}</Text>
+      {!!counter && (
         <Box
-          border={{ side: 'bottom', color: isActive ? 'oasisBlue2' : 'background-contrast-2', size: '2px' }}
-          pad={{ bottom: 'xxsmall', horizontal: 'xxsmall' }}
+          style={{ position: 'absolute', top: '-3px', right: '-5px', zIndex: 2 }}
```

Comment:
`zIndex: 2` displays above mobile navbar and footer, and doesn't even seem to be needed

Reply:
damn it it is needed. will include a fix in the last "change ui" PR

Reply:
hm, when is it needed?
![z](https://user-images.githubusercontent.com/3758846/180828987-f46aa37d-03cd-4a52-b184-431c9d92a7d3.gif)

Reply:
When a link has a counter and next link in DOM hierarchy is active.
In our case: when debonding page is active and Active delegation has a counter.